tests/storage/filesd_metadata_test.py

```diff
@@ -108,14 +109,11 @@
 )
 def storage(request):
     storage, alignment = request.param
-    if not storage.exists():
-        pytest.xfail("{} storage not available".format(storage.name))
-
-    with open(storage.path, "w") as f:
-        f.truncate(0)
-
-    yield Storage(storage.path, storage.sector_size, alignment)
-    oop.stop()
+    with Backend(storage) as backend:
+        with open(backend.path, "w") as f:
+            f.truncate(0)
```

Comment:
The purpose of the backend is to avoid such code. Using the backend enstures that the volume is cleaned up before/after test.